### PR TITLE
Misc code removals/changes

### LINKED ISSUE #3808: The lighting control program keeps a hard-ref to the last mob that used it.

### PR DESCRIPTION
**Pull request by printer16**
Removes some in worlds.
Removes an unused variable or two.
Removed some \red's/blues/greens and updated the travis expected span count.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_powers.dm [hunk 1]
```diff
@@ -229,8 +229,8 @@
 	var/msg = sanitize(input("Message:", "Psychic Whisper") as text|null)
 	if(msg)
 		log_say("PsychicWhisper: [key_name(src)]->[M.key] : [msg]",ckey=key_name(src))
-		M << "\green You hear a strange, alien voice in your head... \italic [msg]"
-		src << "\green You said: \"[msg]\" to [M]"
+		M << "<span class ='notice'>You hear a strange, alien voice in your head... \italic [msg]</span>"
```
Comment (Lohikar) [inline code/modules/mob/living/carbon/human/human_powers.dm]: green != notice

File: code/modules/mob/living/carbon/human/human_powers.dm [hunk 2]
```diff
@@ -229,8 +229,8 @@
...
+		src << "<span class ='notice'>You said: \"[msg]\" to [M]</span>"
```
Comment (Lohikar) [inline code/modules/mob/living/carbon/human/human_powers.dm]: Same here.

File: code/modules/nano/modules/lighting_ctrl.dm [hunk 3]
```diff
@@ -35,7 +32,7 @@
 		update_lighting()
 
 /datum/nano_module/lighting_ctrl/ui_interact(mob/user, ui_key = "main", var/datum/nanoui/ui = null, var/force_open = 1, var/datum/topic_state/state = default_state)
-	lusr = user
+	user.add_hiddenprint(src)
```
Comment (Lohikar) [inline code/modules/nano/modules/lighting_ctrl.dm]: ...I don't think datums can have fingerprints.

**Comment by printer16** [comment]
Updated.

**Pull request merged**